content/en/docs/setup/quickstart.md

```diff
@@ -79,6 +79,29 @@ time="2024-04-25T13:17:41Z" level=info msg="cni config file: /etc/cni/net.d/10-k
 time="2024-04-25T13:17:41Z" level=info msg="command Start cni successful" subsys=manager
 ```
 
+### Mode change
+We provide two modes, ads and workload, for users to choose from. The default mode is currently workload. If we wishes to use the ads mode,  only need to modify a single startup parameter.
+
+```shell
+# Install from Helm
+vim deploy/helm/valuse.yaml
+...
+kmeshDaemonArgs: "--mode=workload --enable-bypass=false --enable-bpf-log=true"
+...
+# change "--mode=workload" into "--mode=ads"
+helm upgrade kmesh ./deploy/helm -n kmesh-system
+
+# Install from Yaml
+vim deploy/yaml/kmesh.yaml
+...
+args: ["./start_kmesh.sh --mode=workload"]
+...
+# change "--mode=workload" into "--mode=ads"
+kubectl apply -f ./deploy/yaml/
```

Comment:
Long term we may stop maintaining the static yamls, so prefer helm install --set deploy.kmesh.containers.kmeshDaemonArgs=